src/app/search/map-friendly-values.pipe.ts

```diff
@@ -112,6 +112,13 @@ export class MapFriendlyValuesPipe implements PipeTransform {
         ['INSTALL', 'Install'],
       ]),
     ],
+    [
+      'openData',
+      new Map([
+        ['0', 'No public test data'],
+        ['1', 'Has public test data'],
```

Comment:
Minor, but should keep the casing the same as the other facets. `Verified` and `Has Checker Workflows` have lower case facets